src/java/org/apache/cassandra/db/compaction/CompactionIterator.java

```diff
@@ -1240,7 +1239,7 @@ private static boolean isAccordTable(ColumnFamilyStore cfs, String name)
 
     private static boolean isAccordJournal(ColumnFamilyStore cfs)
     {
-        return isAccordTable(cfs, AccordKeyspace.JOURNAL);
+        return cfs.getKeyspaceName().equals(SchemaConstants.ACCORD_KEYSPACE_NAME) && cfs.name.startsWith(AccordKeyspace.JOURNAL);
```

Comment:
`== AccordKeyspace.AccordColumnFamilyStores.journal`?

Reply:
We are relying on the fact that each journal in burn test is using its own table, so we separate their compactions, too. I will add a comment that elaborates on this.